samples/tab-device-permissions/nodejs/appPackage/manifest.json

```diff
@@ -1,14 +1,14 @@
 {
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
   "version": "1.0.0",
-  "id": "7ae8f544-94b4-4a54-b5ce-25dd931c7a4a",
+  "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.microsoft.teams.extension",
   "developer": {
     "name": "Teams App, Inc.",
-    "websiteUrl": "https://<<BASE-URL>>",
-    "privacyUrl": "https://<<BASE-URL>>/privacy",
-    "termsOfUseUrl": "https://<<BASE-URL>>/termsofuse"
+    "websiteUrl": "https://www.microsoft.com",
```

Comment:
same here. Is it by design to use www.microsoft.com, or better to use ${{TAB_DOMAIN}}?

Reply:
Fixed